### PR TITLE
fill-eth-data-from-api

### PR DESCRIPTION
**Pull request by jsanchez556**

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# clientdiversity-org-api
+
+This is the repo for <https://clientdiversity.org>, a resource site to assist client diversity efforts. This project is maintained by [Ether Alpha](https://etheralpha.org/).
```
Comment (aguzmant103) [inline api/README.md:3]: geogrpahicdiversity.org

assist geographic diversity efforts

maintained by Ethereum Costa Rica members

File: _data/raw/nodewatch_raw.json
Comment (aguzmant103) [inline _data/raw/nodewatch_raw.json:1]: Is this needed in the PR or can it be removed? (as it'll be autogenerated each time a call happens?)

**Review by aguzmant103 — APPROVED** [review]

File: _data/raw/nodewatch_raw.json
Comment (jsanchez556) [reply _data/raw/nodewatch_raw.json:1]: This file is not needed and can be ignored in gitignore file.

File: api/README.md (hunk 1 above)
Comment (jsanchez556) [reply api/README.md]: Done

**Review by lindsaymoralesb — APPROVED** [review]
lgtm!

**Pull request merged**